packages/flutter/test_fixes/widgets.dart.expect

```diff
@@ -184,4 +185,35 @@ void main() {
 
   // Change made in https://github.com/flutter/flutter/pull/100381
   SelectionOverlay.fadeDuration;
+
+  // Changes made in https://github.com/flutter/flutter/pull/105291
+  BoxDecoration(color: Color(0xFFFF0000), boxShadow: []);
+  BoxDecoration(
+    color: Color(0xFFFF0000),
+    boxShadow: [],
```

Comment:
`BoxDecoration.boxShadow` becomes `ShapeDecoration.shadows`

so the conversion needs to consider that.

Reply:
Is it necessary though? Does this code still work? There is no analysis or lint that would indicate this is broken code.

Reply:
What I think is missing here, is what exactly is failing? Are some of these transformations working, and others not? From what I can see, the fix rules here do not cover all of these cases.

Reply:
> Is it necessary though? Does this code still work? There is no analysis or lint that would indicate this is broken code.

Right. The thing is, `shape` is deprecated but the way we can migrate that is by replacing the `BoxDecoration` with `ShapeDecoration` when the shape is circle. 

What's failing is, the fix is not triggered for BoxShape.circle because we are using `replacedBy`.
![image](https://user-images.githubusercontent.com/20423471/190325895-f440e496-b491-48a5-b64e-8d79f20974d9.png)

If I remove this `replacedBy` change, then every thing works except `BoxDecoration` is not renamed to `ShapeDecoration`. So this might be a bug in `replacedBy`? Or I might doing something wrong.

Reply:
> Is it necessary though

You are right. Fixed!